spec/controllers/sessions_controller_spec.rb

```diff
@@ -63,4 +63,28 @@
       end
     end
   end
+
+  describe 'DELETE#destroy' do
+    it 'renders a redirect response' do
+      delete :destroy
+
+      expect(response).to be_redirect
+    end
+
+    it 'redirects to correct path' do
+      delete :destroy
+
+      expect(response).to redirect_to(root_path)
+    end
+
+    it 'session should be nil' do
```

Comment:
it removes the user_id session

Reply:
Fixed in d19c7b8eb9d2313efb53ac48aa38270a0da94aa8